<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordForm</class>
 <widget class="QDialog" name="RecordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать Запись</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Добавление/Редактирование Записи</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_service_date_time">
       <property name="text">
        <string>Дата и время:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="service_date_time"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_car_id">
       <property name="text">
        <string>Автомобиль:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="car_combo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_employee_id">
       <property name="text">
        <string>Сотрудник:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="employee_combo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_service_id">
       <property name="text">
        <string>Услуга:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="service_combo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="submit_button">
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
